gvsbuild/projects/boringssl.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
             repository="https://github.com/google/boringssl",
             repo_url="https://github.com/google/boringssl.git",
             fetch_submodules=False,
-            tag="f1c75347daa2ea81a941e953f2263e0a4d970c8d",  # commit from master-with-bazel branch
+            tag="44b3df6f03d85c901767250329c571db405122d5",  # commit from master-with-bazel branch
```

Comment:
is the comment still accurate?

Reply:
It seems to: from the history: https://github.com/google/boringssl/commits/44b3df6f03d85c901767250329c571db405122d5
`update master-with-bazel`.

Moreover, it is different from master because of the file structure/build system